### PR TITLE
Related content in main fix

### LINKED ISSUE #6100: Related content shouldn’t sit in the main container/div
**Describe the bug**

The related content component isn’t main content and therefore should not sit within the main landmark. It should sit outside <main role=“main”> like the secondary column components

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to a MAP/STY
2. Inspect the page with developer tools.
3. Find the `main` component.
4. See that related content sits within it.

**Expected behaviour**
For non-main content to not be in the main component.

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #6100 

**Overall change:**

Moved the main landmark to not include related content. 

**Code changes:**

- Removed `main` role from grid component
- added `main` component

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -1352,461 +1352,462 @@ exports[`Story Page should not show the pop-out timestamp when allowDateStamp is
...
-                role="main"
               >
-                <div
-                  class="c3"
-                >
-                  <h1
-                    class="c4"
-                    id="content"
-                    tabindex="-1"
+                <main>
```
Comment (greenc05) [inline src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap]: have we lost the role main attribute here @oluoluoxenfree ?

Comment (oluoluoxenfree) [reply src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap]: @greenc05 I was going off MDN:

> The <main> element has a role of main. Developers should always prefer using the correct semantic HTML element over using ARIA.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role

Comment (oluoluoxenfree) [reply src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap]: You mean it's lost attributes right?

Comment (greenc05) [reply src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap]: We use both ```<main role="main">```, as it was previously please, main is a HTML5 element and we add the role main attribute in addition for AT that does not support HTML5

Comment (greenc05) [reply src/app/pages/StoryPage/__snapshots__/index.test.jsx.snap]: https://github.com/bbc/simorgh/pull/6110/files#diff-67d9f7cbe750226b69381b45f0c9a81dL197

File: src/app/pages/StoryPage/index.jsx [hunk 2]
```diff
@@ -189,14 +189,10 @@ const StoryPage = ({ pageData }) => {
...
-        <Grid
-          item
-          columns={gridColsMain}
-          startOffset={gridOffset}
-          as="main"
-          role="main"
-        >
-          <Blocks blocks={blocks} componentsToRender={componentsToRender} />
+        <Grid item columns={gridColsMain} startOffset={gridOffset}>
+          <main>
```
Comment (mykeels) [inline src/app/pages/StoryPage/index.jsx]: Perhaps, this should be `<main role="main">` as per @greenc05's comment

**Comment by greenc05** [comment]
Happy that related content is not within 'main' and the main html element has a role of main, thanks @oluoluoxenfree I'll leave it to someone else to review how the code is written/tests...

**Review by simonsinclair — COMMENTED** [review]
Hey Olu, nice work. Do you know why the following snapshots have changed, when we've only made a change to the Story Page? Could they have been carried over in error?
- src/app/containers/Bulletin/__snapshots__/index.test.jsx.snap
- src/app/containers/RadioSchedule/Canonical/__snapshots__/index.test.jsx.snap
- src/app/contexts/EventContext/__snapshots__/index.test.jsx.snap

**Comment by oluoluoxenfree** [comment]
@simonsinclair You were correct, needed to pull!

**Review by 12 — APPROVED** [review]
Looks good, thanks for the change 👍

**Review by simonsinclair — APPROVED** [review]
> @simonsinclair You were correct, needed to pull!

Brilliant. Thanks, Olu!

**Comment by sareh** [comment]
We now have integration test snapshots too. This is why CI is failing. Can you please update those with `npm run test:integration -- -u`?

**Comment by paruchurisilpa** [comment]
Looks good to me but looks like the integration tests need to be updated...

**Pull request merged**